### ISSUE TITLE
WRONG FORMAT

### ISSUE THREAD

**Issue by AlexanderEric** [fsareshwala/cs900#1]
NO SUCH THINGS AS .CC if you use that kind of languages structure

**Comment by fsareshwala** [comment]
Thanks for bringing this up. I'm confused what you are referring to. Can you rephrase the issue or be more specific as to what the concern is?

**Comment by fsareshwala** [comment]
Question is ambiguous and poorly formatted.

Re-open if you think this is necessary.

**Comment by AlexanderEric** [comment]
Ok, you wrongly save file as .cc, .cc file are not a reable format. 
If you write using c format, save as .dll
Do not create another .c format, which is .in format, save as .c

**Comment by fsareshwala** [comment]
The `.cc` file extension is a completely valid extension to use for C++. They are plain text files and should be editable by any text editor. If you are unable to open them on your computer, you can update your text editor file mappings to allow you to open them.
